presto-spi/src/main/java/com/facebook/presto/spi/ColumnHandle.java

```diff
@@ -13,6 +13,13 @@
  */
 package com.facebook.presto.spi;
 
+import com.facebook.presto.spi.type.NestedField;
+
+import java.util.Optional;
+
 public interface ColumnHandle
 {
+    default void setNestedField(Optional<NestedField> field) {}
```

Comment:
ColumnHandle is meant to be like an opaque identifier for columns. They should not expose any "behavior".